tutorials/AFQ_publish_tutorials.m

```diff
@@ -0,0 +1,36 @@
+function AFQ_publish_tutorials(remotename)
+% Publish all the AFQ tutorials as html pages push to gh-pages
+%
+% AFQ_publish_tutorials(remotename)
+%
+% inputs:
+% remotename = string for a git remote repository. Default is 'origin'
+%
+% Written by JDY, Feb, 2016
+
+if ~exist('remotename', 'var') || isempty(remotename)
+    remotename = 'origin';
+end
+afqdir = AFQ_directories;
+tdir = fullfile(afqdir,'tutorials');
+
+% List of tutorials to publish
+tlist = {'AFQ_example.m'};
+
+% Publish tutorial list
+opts.format = 'html';
+opts.outputDir = fullfile(tdir,'htmltmp');
+for ii = 1:length(tlist)
+    publish(tlist{ii},opts); 
```

Comment:
Does `publish` always put the files in the same place? That is, does this work well, when run not from within the AFQ directory? Otherwise, you could move the `cd` from line 29 to before this loop.